src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -672,3 +672,52 @@ PARAM_DEFINE_INT32(MPC_OBS_AVOID, 0);
  * @group Mission
  */
 PARAM_DEFINE_INT32(MPC_YAW_MODE, 0);
+
+/**
+ * Enable weathervane for manual.
```

Comment:
These can live in the weathervane library.

Reply:
Yes